catalogs/catalog-jdbc-starrocks/src/main/java/org/apache/gravitino/catalog/starrocks/converter/StarRocksTypeConverter.java

```diff
@@ -18,19 +18,137 @@
  */
 package org.apache.gravitino.catalog.starrocks.converter;
 
-import org.apache.commons.lang3.NotImplementedException;
 import org.apache.gravitino.catalog.jdbc.converter.JdbcTypeConverter;
 import org.apache.gravitino.rel.types.Type;
+import org.apache.gravitino.rel.types.Types;
 
 /** Type converter for StarRocks. */
 public class StarRocksTypeConverter extends JdbcTypeConverter {
+
+  static final String BIGINT = "bigint";
+  static final String BOOLEAN = "boolean";
+  static final String DECIMAL = "decimal";
+  static final String DOUBLE = "double";
+  static final String FLOAT = "float";
+  static final String INT = "int";
+  static final String LARGEINT = "largeint";
+  static final String SMALLINT = "smallint";
+  static final String TINYINT = "tinyint";
+
+  static final String BINARY = "binary";
+  static final String VARBINARY = "varbinary";
+  static final String CHAR = "char";
+  static final String STRING = "string";
+  static final String VARCHAR = "varchar";
+
+  static final String DATE = "date";
+  static final String DATETIME = "datetime";
+
+  static final String ARRAY = "array";
+  static final String JSON = "json";
+  static final String MAP = "map";
+  static final String STRUCT = "struct";
+
+  static final String BITMAP = "bitmap";
+  static final String HLL = "hll";
+
+  static final String BIT = "BIT";
```

Comment:
Are these types not yet supported?

Reply:
Yes, those type are not supported for now.
Because `JdbcTypeBean` doesn't have element type

Reply:
I see.